app/models/ability.rb

```diff
@@ -34,8 +34,8 @@ def not_signed_in
       conference.splashpage && conference.splashpage.public == true
     end
     # Can view the schedule
-    can [:schedule, :events], Conference do |conference|
-      conference.program.cfp && conference.program.schedule_public
+    can [:show, :schedule, :events], Conference do |conference|
```

Comment:
previously `schedule` was a method in `conferences_controller`, but now it is a different controller